### PR TITLE
fix(source): return isFetching  and isReady with false by default in get

### PR DESCRIPTION
**Pull request by DAreRodz**

### CODE REVIEW

**Comment by luisherranz** [comment]
@DAreRodz is there a test to assert that reactivity for `isReady` and `isFetching` works? Something like this:

```js
test("...", done => {
  expect(state.source.get("/").isReady).toBe(false);
  observe(() => {
    if (state.source.get("/").isReady === true) done();
  });
  actions.source.fetch("/");
});

```

**Comment by DAreRodz** [comment]
There is not, but it should. I will add it.

File: packages/mars-theme/src/components/theme.js [hunk 1]
```diff
@@ -38,7 +32,12 @@ const Theme = ({ state }) => {
       <HeadContainer>
         <Header />
       </HeadContainer>
-      <Body>{Content}</Body>
+      <Body>
+        {data.isFetching && <Loading />}
```
Comment (luisherranz) [inline packages/mars-theme/src/components/theme.js:36]: Wow, I love that! So simple :)

**Review by orballo — APPROVED** [review]

**Comment by orballo** [comment]
I guess this one needs a squash, I'll let you to it @DAreRodz 😂

**Pull request merged**